gnat2goto/driver/tree_walk.adb

```diff
@@ -694,31 +694,61 @@ package body Tree_Walk is
       Resolved_Underlying : constant Irep :=
         Follow_Symbol_Type (Underlying, Global_Symbol_Table);
       --  ??? why not get this from the entity
+
+      function Get_Bound_Symbol (Bound_Node : Node_Id) return Bound_Type_Symbol
+        with Pre => Get_Attribute_Id (Attribute_Name (Bound_Node))
+          in Attribute_First | Attribute_Last;
+      function Get_Bound_Symbol (Bound_Node : Node_Id) return Bound_Type_Symbol
+      is
+      begin
+         if Get_Attribute_Id (Attribute_Name (Bound_Node)) = Attribute_First
+         then
+            return Bound_Type_Symbol (Do_Array_First (Bound_Node));
+         else
+            return Bound_Type_Symbol (Do_Array_Last (Bound_Node));
+         end if;
+      end Get_Bound_Symbol;
+
+      Lower_Bound : constant Node_Id := Low_Bound (Range_Expr);
+      Upper_Bound : constant Node_Id := High_Bound (Range_Expr);
+
+      Lower_Bound_Value : Integer;
+      Upper_Bound_Value : Integer;
+
+      Result_Type : constant Irep := New_Irep (I_Bounded_Signedbv_Type);
    begin
       if not (Kind (Resolved_Underlying) in Class_Bitvector_Type) then
          Report_Unhandled_Node_Empty (Range_Expr, "Do_Base_Range_Constraint",
```

Comment:
why not return `Report_Unhandled_Node_Type`?

Reply:
Done.